prow/cmd/status-reconciler/main.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"k8s.io/test-infra/pkg/flagutil"
 	"k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/config/secret"
 	prowflagutil "k8s.io/test-infra/prow/flagutil"
 	_ "k8s.io/test-infra/prow/hook"
```

Comment:
@cjwagner @stevekuznetsov  any idea why we need this?

Reply:
Because we need to import all plugins to actually load the plugin config. If you don't import the plugins we fail validation on plugin config loading as we don't find the plugin names registered.